ui/package.json

```diff
@@ -43,155 +43,155 @@
     "vault:cluster": "VAULT_REDIRECT_ADDR=http://127.0.0.1:8202 vault server -log-level=error -dev -dev-root-token-id=root -dev-listen-address=127.0.0.1:8202 -dev-ha -dev-transactional"
   },
   "devDependencies": {
-    "@babel/cli": "^7.24.6",
-    "@babel/core": "^7.26.10",
-    "@babel/eslint-parser": "^7.22.15",
-    "@babel/plugin-proposal-class-properties": "^7.18.6",
-    "@babel/plugin-proposal-decorators": "^7.23.2",
-    "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
-    "@babel/plugin-transform-block-scoping": "^7.12.1",
-    "@babel/preset-env": "^7.24.6",
-    "@babel/preset-typescript": "^7.24.6",
-    "@docfy/ember": "^0.8.5",
-    "@ember/legacy-built-in-components": "^0.4.1",
-    "@ember/optional-features": "^2.0.0",
-    "@ember/render-modifiers": "^1.0.2",
-    "@ember/string": "^3.1.1",
-    "@ember/test-helpers": "^3.2.0",
-    "@ember/test-waiters": "^4.0.0",
-    "@glimmer/component": "^1.1.2",
-    "@glimmer/tracking": "^1.1.2",
-    "@icholy/duration": "^5.1.0",
-    "@lineal-viz/lineal": "^0.5.1",
-    "@tsconfig/ember": "^2.0.0",
-    "@types/codemirror": "^5.60.15",
-    "@types/d3-array": "^3.2.1",
-    "@types/ember-data": "^4.4.16",
-    "@types/qunit": "^2.19.4",
-    "@types/rsvp": "^4.0.4",
-    "@types/shell-quote": "^1.7.1",
-    "@typescript-eslint/eslint-plugin": "^5.19.0",
-    "@typescript-eslint/parser": "^5.19.0",
-    "asn1js": "^2.2.0",
-    "autosize": "^4.0.0",
-    "babel-plugin-inline-json-import": "^0.3.2",
-    "base64-js": "^1.3.1",
-    "broccoli-asset-rev": "^3.0.0",
+    "@babel/cli": "~7.27.0",
```

Comment:
Now different than babel core bumping this to match what was being resolved in yarn.lock. 
![image](https://github.com/user-attachments/assets/7698647e-1795-44b2-85a1-41e56eefbbff)